HorribleDownloader/cmd.py

```diff
@@ -280,12 +283,16 @@ def should_download(episode):
             logger.info("user has aboorted the download")
             exit(1)
 
-
-    for episode in downloads_list:
-        download(episode, qualities, config.download_dir)
-        config.update_entry(episode["title"], episode["episode"])
-        logger.info(f'updated entry: {episode["title"]} - {episode["episode"]}')
+    if args.export:
+        for episode in downloads_list:
```

Comment:
Same goes here, you must iterate over the qualities.

Note: update the entry once all qualities were printed out.